modules/cluster-logging-updating-logging.adoc

```diff
@@ -5,27 +5,29 @@
 [id="cluster-logging-updating-logging_{context}"]
 = Updating cluster logging
 
-After updating the {product-title} cluster, you can update cluster logging from 4.3 to 4.4 by updating the subscription for the Elasticsearch Operator and the Cluster Logging Operator.
+After updating the {product-title} cluster, you can update cluster logging from 4.4 to 4.5 by updating the subscription for the Elasticsearch Operator and the Cluster Logging Operator. You must update the Elasticsearch Operator before updating the Cluster Logging Operator and must update both the Elasticsearch Operator and the Cluster Logging Operator at the same time. Kibana is unusable when the Elasticsearch Operator has been upgraded but the Cluster Logging Operator has not been upgraded.
```

Comment:
this should suggest "should" upgrade EO first.  it's not a must

Reply:
based on https://bugzilla.redhat.com/show_bug.cgi?id=1850339 it looks like it should be a must

Reply:
if we do keep it as "should" we likely need to tell users they will need to bounce their CLO pod afterwards so that it can process the type "Kibana"